### DISCUSSION TITLE
can prism create a .mrpack ?

### DISCUSSION THREAD

**Discussion by Laurojjj** [PrismLauncher/PrismLauncher#2902] [Q&A] [answered]
i want to create an mod pack thats why i am asking

**Comment by Ryex** [comment, marked as answer]
It absolutely can

Right click an instance > export > Modrinth
![image](https://github.com/user-attachments/assets/b55f6111-0f69-4729-b0be-24a937949a26)


Or  Select an instance > Sidebar > export (down arrow) > Modrinth
![image](https://github.com/user-attachments/assets/2d608b30-b5da-4a92-8a9a-1040aad4da5e)

**Reply by Laurojjj** [reply]
omg thank youu
